src/platform/silabs/platformAbstraction/WiseMcuSpam.cpp

```diff
@@ -36,8 +36,15 @@ extern "C" {
 #include "sl_event_handler.h"
 #include "sl_si91x_button.h"
 #include "sl_si91x_button_pin_config.h"
+#ifndef SI917_DEVKIT
```

Comment:
This define doesn't reflect what it represents. It should be something along the lines for `SL_MATTER_ENABLE RGB`.

Reply:
This macro SI917_DEVKIT is defined in wifi_sdk and used the same in gn build, so that again in slc releases need to maintain the define for BRD2605A board.